DataCollection/src/Services/Models/Survey/WaterbirdForagingListItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace FlightNode.DataCollection.Services.Models.Survey
+{
+    /// <summary>
+    /// Use for displaying a compact list of waterbird foraging survey results.
+    /// </summary>
+    public class WaterbirdForagingListItem
```

Comment:
When displaying a list of surveys on the screen, I think we only need to show these few fields.

`StartDate` is a string so that it can say "missing" when the survey is pending and the user hasn't filled in that information.